src/convertToObject.js

```diff
@@ -3,16 +3,39 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrWithoutSemi = sourceString
+    .split('')
+    .filter(el => el !== ';');
+
+  const arrOfProp = arrWithoutSemi
+    .join('')
+    .split('\n')
```

Comment:
what does it mean `.split('\n')` here?

Reply:
you can firstly split by `;` and then by `:`